src/test/java/one/util/streamex/TestHelpers.java

```diff
@@ -182,6 +325,18 @@ static void repeat(int times, IntConsumer consumer) {
         }
     }
 
+    static void intStreamEx(Supplier<IntStream> base, Consumer<IntStreamExSupplier> consumer) {

```

Comment:
I think it's easier to delegate to streamEx() without defining any additional types. Like `streamEx(() -> base.get().boxed(), s -> consumer.accept(s.get().mapToInt(x -> x)))`. Should be moreless similar but without much code duplication.